src/containers/pages/FocusInvestigation/active/tests/index.test.tsx

```diff
@@ -485,6 +433,27 @@ describe('containers/pages/ActiveFocusInvestigation', () => {
     await new Promise(resolve => setImmediate(resolve));
     wrapper.update();
 
+    expect(props.supersetService).toHaveBeenCalledTimes(1);
+    expect(props.supersetService).toHaveBeenCalledWith(0, {
```

Comment:
Is it possible to get such an assertion that shows the userPlanIds explicitly in use?

Reply:
yes